solana/tests/utils/tbr-wrapper.ts

```diff
@@ -81,99 +81,162 @@ export class TbrWrapper {
     owner: PublicKey;
     feeRecipient: PublicKey;
     admins: PublicKey[];
-  }): Promise<TransactionSignature> {
-    return $.sendAndConfirm(await this.client.initialize(args), this.signer);
+  }): Promise<VersionedTransactionResponse | null> {
+    return $.getTransaction(
+      await $.sendAndConfirm(await this.client.initialize(args), this.signer),
+    );
```

Comment:
This common `getTransaction(sendAndConfirm( ..., this.signer))` call structure should preferably go into private function that just takes a promise to cut down on the boiler-plate.

Reply:
`getTransaction` now accepts a promise as well.